packages/cli/src/lib/platform/common.ts

```diff
@@ -71,3 +74,31 @@ export const setPlatformDefaults = async () => {
         return false;
     }
 };
+
+const profileConfig = profileManager.getProfileConfig();
+const platformRequirementsValid = (
+    program: Command & { _helpShortFlag?: any, _helpLongFlag?: any },
+    token: string,
+    env: configEnv,
+    middlewareApiUrl: string
+) =>
+    token &&
+    isProductionEnv(env) &&
+    middlewareApiUrl &&
+    !process.argv.includes(program._helpShortFlag) &&
+    !process.argv.includes(program._helpLongFlag);
+
+export const initPlatform = async (program: Command) => {
```

Comment:
I would add extra check, just in case:
`if (!isProductionEnv(profileConfig.env)) return;`

Reply:
I think this depends on which API we're calling - if we want to use topics on local, yes, but if we want to list hubs, that may be different.

I'll look into this.

Reply:
implemented @ErykSol solution, resolving all conversations